integration_tests/tests/forseti/controls/inventory.rb

```diff
@@ -0,0 +1,122 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kms_resources_names = attribute('kms_resources_names')
+
+require 'json'
+control 'inventory' do
+
+    describe "Inventory" do
+        before :context do
+            command("forseti inventory purge 0").result
+            command("forseti inventory create").result
+
+             # This variable cannot be used after all the inventories have been purged..
+            @inventory_id = JSON.parse(command("forseti inventory list").stdout).fetch("id")
+        end
+
+        describe "Create an inventory" do
```

Comment:
As discussed, inventory needs to be created once for all the tests instead of creating for each and every test. Hence, `inventory create` command needs to be in the before section. The description is aptly named `Create an inventory` as successful creation of inventory is checked in this test by asserting against `SUCCESS` and by performing the database checks.